### PR TITLE
Update restore-statements-headeronly-transact-sql.md for NULLable fie…

### PR DESCRIPTION
**Pull request by kmieciu2003**
…lds marker

### CODE REVIEW

**Comment by Jak-MS** [comment]
@MikeRayMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/statements/restore-statements-headeronly-transact-sql.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ FROM <backup_device>
 |Column name|Data type|Description for SQL Server backup sets|  
 |-----------------|---------------|--------------------------------------------|  
 |**BackupName**|**nvarchar(128)**|Backup set name.|  
-|**BackupDescription**|**nvarchar(255)**|Backup set description.|  
+|**BackupDescription**|**nvarchar(255) NULL**|Backup set description.|
```
Comment (rwestMSFT) [inline docs/t-sql/statements/restore-statements-headeronly-transact-sql.md]:
```suggestion
|**BackupDescription**|**nvarchar(255)** NULL|Backup set description.|  

```

File: docs/t-sql/statements/restore-statements-headeronly-transact-sql.md [hunk 2]
```diff
@@ -86,7 +86,8 @@ FROM <backup_device>
 |Column name|Data type|Description for SQL Server backup sets|  
 |-----------------|---------------|--------------------------------------------|  
 |**BackupName**|**nvarchar(128)**|Backup set name.|  
-|**BackupDescription**|**nvarchar(255)**|Backup set description.|  
+|**BackupDescription**|**nvarchar(255)** NULL|Backup set description.|  
+
```
Comment (rwestMSFT) [inline docs/t-sql/statements/restore-statements-headeronly-transact-sql.md]:
```suggestion
```

**Comment by rwestMSFT** [comment]
Thanks for your contribution, @kmieciu2003. This article needs to be refreshed, so we will take your contribution into account.

**Pull request closed without merging**